privaterelay/templates/faq.html

```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
           What about spam?
         </h2>
         <p class="faq-answer">
-          We use Amazon SES as our Email Service Provider, which enforces industry standard spam controls. We also have the ability to enforce even more strict spam filtering.      </p>
+          While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them. If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
```

Comment:
We could make "report this to us" go to our GitHub issues? Or maybe the AMO page for reviews?

Reply:
Your call -  I think GitHub may make more sense, but AMO might be easier for people unfamiliar with GitHub

Reply:
Let's do AMO. Monitor users aren't likely to know GitHub well enough to file issues.